planner/core/logical_plan_builder.go

```diff
@@ -310,8 +310,17 @@ func (p *LogicalJoin) extractOnCondition(conditions []expression.Expression, der
 	return
 }
 
+// This function will return the table alias of a given logical plan.
+// If there're multiple tables, it will return nil.
+// This function will be further developed in the future.
```

Comment:
This line provides little info and makes me curious, you can add more details like what will be developed and why.

Reply:
I have rewrite the comments... I find out it may only be influenced by Schema refactoring.